packages/build/src/download-center/config.ts

```diff
@@ -166,17 +170,30 @@ export function createVersionConfig(
   publicArtifactBaseUrl: string = ARTIFACTS_URL_PUBLIC_BASE
 ) {
   const { version } = packageInformation('linux-x64').metadata;
-  return {
-    _id: version,
-    version: version,
-    platform: ALL_PACKAGE_VARIANTS.map((packageVariant: PackageVariant) => ({
+  const platformMap: Map<string, PlatformWithPackages> = new Map();
+
+  for (const packageVariant of ALL_PACKAGE_VARIANTS) {
+    const platformName = getDownloadCenterDistroDescription(packageVariant);
+    const currentPlatform = platformMap.get(platformName) || {
       arch: getArch(packageVariant),
-      os: getDistro(packageVariant),
-      name: getDownloadCenterDistroDescription(packageVariant),
+      os: getDownloadCenterDistroDescription(packageVariant),
+      packages: { links: [] },
+    };
+
+    currentPlatform.packages.links.push({
+      name: getDownloadCenterPackageType(packageVariant),
       download_link:
         publicArtifactBaseUrl +
         getPackageFile(packageVariant, packageInformation).path,
-    })),
+    });
+
+    platformMap.set(platformName, currentPlatform);
+  }
+
+  return {
+    _id: version,
+    version: version,
+    platform: [...platformMap.values()],
   } as const;
```

Comment:
```suggestion
  };
```

should let you get rid of the `as any` casts above